code/controllers/Processes/power.dm

```diff
@@ -1,14 +1,30 @@
 var/global/list/power_machinery_profiling = list()
 var/global/list/power_machines = list()
+var/global/list/datum/powernet/powernets = list() //Holds all powernet datums in use or pooled
+var/global/list/cable_list = list() //Index for all cables, so that powernets don't have to look through the entire world all the time
 
 /datum/controller/process/power_machinery
-	schedule_interval = 37 // every 2 seconds
+	schedule_interval = 17 // every 2 seconds
```

Comment:
Funny you should mention it because it's near a change in this commit

The comment for fast machines is "0.6 seconds" (CF is changing it) but it actually takes 7 ticks